index.html

```diff
@@ -83,135 +77,135 @@ <h2>Tonic</h2>
+                <div >
+                    <div >
                         <h2>Multi-Post Stories</h2>
-                        <ul class="copyright">
-                            <li class="m-img">CANOPY</li>
-                            <li class="d-img">FACEBOOK</li>
-                            <li class="m-img">Back End Dev</li>
-                            <li class="d-img">Full Stack Dev</li>
+                        <ul >
+                            <li >CANOPY</li>
+                            <li >FACEBOOK</li>
+                            <li >Back End Dev</li>
+                            <li >Full Stack Dev</li>
                             <li>2015</li>
                         </ul>
                     </div>       
-                    <p class="m-img">
+                    <p >
                         A daily selection of privately personalized reads; 
                         no accounts or sign-ups required.
                     </p>
-                    <p class="d-img">
+                    <p >
                         Experimental content creation feature that allows 
                         users to add to an existing story over the course 
                         of a day without spamming their friends.
                     </p>
-                    <ul class="tags">
+                    <ul >
                         <li><a href="#">HTML</a></li>
-                        <li class="d-img"><a href="#">Ruby on rails</a></li>
+                        <li ><a href="#">Ruby on rails</a></li>
                         <li><a href="#">CSS</a></li>
                         <li><a href="#">JavaScript</a></li>     
                     </ul>
-                    <div class="btn-container">
-                        <div class="btn"><a href="https://github.com/sagieramos">See Project</a></div>
+                    <div >
+                        <div ><a href="https://github.com/sagieramos">See Project</a></div>
                     </div>
             </section>
             <!-- Card Three -->
-            <section class="card">
-                <img class="m-img" src="./assets/cards/card3.svg" alt="Tonic project">
-                <img class="d-img" src="./assets/cards/card1.svg" alt="Tonic project">
-                <div class="card-container">
-                    <div class="card-header">
-                        <h2 class="m-img">Tonic</h2>
-                        <h2 class="d-img">Facebook 360</h2>
-                        <ul class="copyright">
-                            <li class="m-img">CANOPY</li>
-                            <li class="d-img">FACEBOOK</li>
-                            <li class="m-img">Back End Dev</li>
-                            <li class="d-img">Full Stack Dev</li>
+            <section >
+                <img  src="./assets/cards/card3.svg" alt="Tonic project">
+                <img  src="./assets/cards/card1.svg" alt="Tonic project">
```

Comment:
- I saw in every open tag has a space in the end and I think you need to delete it because it makes your code looks unprofessional.